tests/test_datayaml.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import pytz
+import yaml
+
+from datetime import datetime
+
+
+KEYS_CONTAINING_DATES = ['last_updated', 'expires', 'sent']
+
+
+def get_alerts_data(path):
+    with open(path) as file:
+        alerts_data = yaml.load(file, Loader=yaml.FullLoader)
+    return alerts_data
+
+
+def does_date_contain_explicit_offset(date):
+    if not isinstance(date, datetime):
+        return False
+    localised_datetime = pytz.timezone('Europe/London').localize(date.replace(tzinfo=None))
+    return localised_datetime.utcoffset() == date.utcoffset()
+
+
+@pytest.mark.parametrize('yaml_file_path', [
+    'tests/test_files/example_alert_data.yaml',  # example alert that shows how dates should be formatted
```

Comment:
We don't need a separate example file. The "example" mentioned in the card was meant to be a comment in `data.yaml`, with guidance to change the offset when copy/pasting. The assertion here is even better, because we actually check for the expected offset. Soon, we will also have real alerts to copy from.

It might be easier to split this in two:

- One test for the actual data file.

- One test function for the assertion code, which could mock the YAML load bit to test a few different pieces of data. Or we could just trust that the assertion code works. If we do want to test the test 😄, then we should have some test data that we expect to fail as well.

Reply:
ok,  I will get rid of test data, I don't think we need to test the test 😅